.github/workflows/main.yml

```
@@ -62,41 +62,41 @@ jobs:
       - run: npm run build
 
       # Pack
-      - run: npm run pack
-      - uses: actions/upload-artifact@v4
-        with:
-          name: packages
-          path: |
-            packages/**/*.tgz
-            packages/**/*.nupkg
-            angular-workspace/**/*.tgz
-          if-no-files-found: error
-      - uses: actions/upload-pages-artifact@v3
-        with:
-          path: 'packages/site/dist'
+      # - run: npm run pack
+      # - uses: actions/upload-artifact@v4
+      #   with:
+      #     name: packages
+      #     path: |
+      #       packages/**/*.tgz
+      #       packages/**/*.nupkg
+      #       angular-workspace/**/*.tgz
+      #     if-no-files-found: error
+      # - uses: actions/upload-pages-artifact@v3
+      #   with:
+      #     path: 'packages/site/dist'
 
       # Publish to Chromatic (from any push)
       - name: Publish to Chromatic
         if: env.HAS_CHROMATIC_PROJECT_TOKEN == 'true' && github.event_name == 'push'
-        uses: chromaui/action@v11
+        uses: chromaui/action@v10
         with:
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
           onlyChanged: "!startsWith(github.ref, 'refs/heads/main')" # Use TurboSnap for PR builds
```

A good trigger all test to emulate running on main is to flip this condition
```suggestion
          onlyChanged: "startsWith(github.ref, 'refs/heads/main')" # Use TurboSnap for PR builds
```